_dev/src/components/google-ads-account/google-ads-account-popin-new.vue

```diff
@@ -200,6 +200,8 @@
       slot="modal-footer"
       v-else-if="stepActiveData === 3"
     >
+      <BannerAds :is-banner="isBanner" />
```

Comment:
```suggestion
      <BannerAds :is-banner="false" />
```

Should work as well (and avoids the use of a new data variable)